assets/sass/style.sass

```diff
@@ -0,0 +1,119 @@
+@charset "utf-8"
+{{ $extraColors := site.Params.colors.extra }}
+{{ $fontAwesomeVersion := site.Params.font_awesome_version }}
+{{ $fonts := site.Params.fonts }}
+{{ if $fonts }}
+{{ $fontSlice := (slice) }}
+{{ range $fonts }}
+{{ $fontSlice = $fontSlice | append (printf "%s:%s" (replace .name " " "+") (delimit .sizes ",")) }}
+{{ end }}
+{{ $fontsUrl := printf "https://fonts.googleapis.com/css?family=%s" (delimit $fontSlice "|") }}
+@import url({{ $fontsUrl }})
+{{ end }}
+
+@import url({{ $fontAwesomeVersion }})
```

Comment:
I think this line might be the culprit, leading to the page trying to download `/css/5.12.0` during every load, which of course doesn't exist.